boards/components/src/process_console.rs

```diff
@@ -143,15 +143,15 @@ impl<const COMMAND_HISTORY_LEN: usize, A: 'static + Alarm<'static>> Component
         // pointers to them.
         let kernel_addresses = unsafe {
             process_console::KernelAddresses {
-                stack_start: &_sstack as *const u8,
-                stack_end: &_estack as *const u8,
```

Comment:
I tried to remove the unsafe here, but the compiler still insists that this is a _usage_ of an extern static. I think this is due to the implementation of a macro, which uses an experimental syntax that still triggers the safety lint.